src/createStore.js

```diff
@@ -66,9 +66,13 @@ export default function createStore(reducer, initialState) {
         return
       }
 
-      isSubscribed = false
-      var index = listeners.indexOf(listener)
-      listeners.splice(index, 1)
+      listeners = listeners.filter(x => {
```

Comment:
Filter just looks more clean to me. I can change it to concat + slice that also fine.